src/utilities/idf/Test/ExtensibleGroup_GTest.cpp

```diff
@@ -488,17 +491,19 @@ TEST_F(IdfFixture, ExtensibleGroup_Erase) {
   ASSERT_FALSE(eg.empty());  // Brick, Insulation, Insulation, Gypsum
 
   // erase and check pointers
-  EXPECT_EQ(static_cast<unsigned>(2), temp[2].numSources());
+  EXPECT_EQ(static_cast<unsigned>(2), wsInsulation.numSources());
   values = wsConstruction.eraseExtensibleGroup(1);
   ASSERT_EQ(static_cast<unsigned>(1), values.size());
-  EXPECT_EQ("Insulation", values[0]);  // Brick, Insulation, Gypsum
-  EXPECT_EQ(static_cast<unsigned>(1), temp[2].numSources());
+  EXPECT_EQ(openstudio::toString(wsInsulation.handle()), values[0]);  // Brick, Insulation, Gypsum
```

Comment:
Adjust existing test since return of eraseExtensibleGroup(index) for **WorkspaceObject** changes from object name to handle string for source fields. I don't think we care about this change.